### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by raynelle03**
Any suggestions on the UI , strings please shout

Simulator Screen Shot - iPhone 11 - 2020-11-17 at 10 54 13

Simulator Screen Shot - iPhone 11 - 2020-11-17 at 10 56 59

### CODE REVIEW

File: iosHackthonApp/iosHackthonApp/Login/LoginView.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		coloredNavAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor.white]
+		UINavigationBar.appearance().standardAppearance = coloredNavAppearance
+		UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
+	}
+
+	var body: some View {
+		NavigationView {
+			ScrollView {
+				VStack {
+					Image("") // Add proper logo
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/Login/LoginView.swift]: Pretty picky I know, but can we add TODO to the beginning of your comment to make it really easy to search for them in the project please

File: iosHackthonApp/iosHackthonApp/Login/LoginView.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+						Text(Strings.Login.email)
+						TextField(Strings.Login.emailPlaceholder, text: $email)
+							.modifier(GreyTextFieldStyle())
+						Text(Strings.Login.password)
+						SecureField(Strings.Login.passwordPlaceholder, text: $password)
+							.modifier(GreyTextFieldStyle())
+						Spacer()
+
+						GeometryReader { geometry in
+							let backgroundColor = email.isEmpty || password.isEmpty ? ColorManager.gray: ColorManager.appPrimary
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/Login/LoginView.swift]: lets shift the .isEmpty checking to a computed variable and then we can use it for both the background colour and whether the button is disabled

File: iosHackthonApp/iosHackthonApp/Login/LoginView.swift [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+						print("Go to sign up")
+					}) {
+						Text(Strings.Login.signupButtonTitle)
+							.frame(alignment: .center)
+							.foregroundColor(ColorManager.appPrimary)
+					}
+				}
+			}
+		}
+		.navigationViewStyle(StackNavigationViewStyle())
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/Login/LoginView.swift]: I removed this from the other places it was.  It was in some files and not others and all it is for is making the view on iPad the full screen, which AFAIK we haven't really considered how things are on the iPad.

File: iosHackthonApp/iosHackthonApp/Login/LoginView.swift (hunk 1 above)
Comment (raynelle03) [reply iosHackthonApp/iosHackthonApp/Login/LoginView.swift]: No worries..added a TODO

File: iosHackthonApp/iosHackthonApp/Login/LoginView.swift (hunk 3 above)
Comment (raynelle03) [reply iosHackthonApp/iosHackthonApp/Login/LoginView.swift]: Got rid of it

File: iosHackthonApp/iosHackthonApp/Login/LoginView.swift (hunk 2 above)
Comment (raynelle03) [reply iosHackthonApp/iosHackthonApp/Login/LoginView.swift]: Nice catch..Done

**Review by KaneCheshire — APPROVED** [review]

**Review by Gmiller290488 — APPROVED** [review]

**Pull request merged**